lua/telescope/_extensions/file_browser/make_entry.lua

```diff
@@ -157,6 +157,16 @@ local make_entry = function(opts)
       table.insert(widths, { width = strings.strdisplaywidth(icon) })
       table.insert(display_array, { icon, icon_hl })
     end
+
+    local git_status, _ = Job:new({ cwd = opts.cwd, command = "git", args = { "status", "--short", "--", path_display } }):sync()
```

Comment:
Just a general comment in case you'd contribute more in the future to telescope. `path_display` can change tremendously depending on user config. In context of file-browser `entry.value` denotes absolute paths.

Reply:
Ah good to know!

I was struggling with getting the correct git status but in the end it was because I was in the wrong working directory and not because of the path. So the relative path (that I'm using the `display_path` here as) shouldn't be needed anymore.